rero_ils/modules/stats/cli.py

```diff
@@ -66,25 +67,32 @@ def collect(type):
         _stats = StatsForLibrarian(to_date=to_date)
     else:
         return
-    stat = Stat.create(
-            dict(type=type, date_range=date_range, values=_stats.collect()),
-            dbcommit=True, reindex=True)
-    click.secho(
-        f'Statistics of type {stat["type"]} have been collected and created.\
-        New pid: {stat.pid}', fg='green')
+
+    with current_app.app_context():
```

Comment:
This will not solve the db timeout, you need to compute the collect outside of the app_context,